.travis.yml

```diff
@@ -28,7 +28,7 @@ matrix:
   include:
     - node_js: 6
       env: TEST_SUITE=node-canvas-prebuilt
-      script: "export CXX=g++-4.8 && npm i canvas-prebuilt && npm test"
+      script: "export CXX=g++-4.8 && yarn i canvas-prebuilt && yarn test"
```

Comment:
`yarn i` doesn't work, non of the shorthands are carried over from npm.

`yarn install` can be just `yarn` though, FWIW

Reply:
Here it's `yarn add` as well